zowe-rest-api-sample-spring/docs/zowe-integrate-with-apiml.md

```diff
@@ -74,6 +74,59 @@ It has two sections:
 
 The description of the properties are in the Zowe documentation.
 
+## Integration with the Zowe in Docker on Your Computer
```

Comment:
Probably should describe that it's integration of the sample API service with Zowe API ML in Docker (and so throughout).

I think "Zowe" is broad and might confuse zlux.